CONTRIBUTING.md

```diff
@@ -96,34 +87,41 @@ Before you start working on a larger contribution, you should get in touch with
     git push origin my-fix-branch
     ```
 
-* In GitHub, send a pull request to `firebase-js-sdk:master`.
-* Add changeset. See [Adding changeset to PR](#adding-changeset-to-pr)
-* All pull requests must be reviewed by a member of the Firebase JS SDK team, who will merge it when/if they feel it is good to go.
+* In GitHub, create a pull request against the `firebase-js-sdk:master` branch.
+* Add changeset. See [Adding changeset to PR](#adding-changeset-to-pr).
+* All pull requests must be reviewed by a member of the Firebase JS SDK team, who will merge it
+when/if they feel it is good to go.
 
 That's it! Thank you for your contribution!
 
 #### Adding changeset to PR
-Every PR that would trigger a release should include a changeset file. To make
-this process easy, a message will be sent to every PR with a link that you can
-click to add changeset files in the Github UI directly.
-[Example message](https://github.com/firebase/firebase-js-sdk/pull/3284#issuecomment-649718617).
+The repository uses changesets to associate PR contributions with major and minor version releases.
+If your change is a feature or a behavioral change (either of which should correspond to a version
+bump) then you will need to generate a Changeset in your PR to track the change.
 
-#### What to include in the changset file
+Start the changeset creation process by running the following command in the base directory of the
+repository:
 
-You should include the version bump for your package as well as the description
-for the change. Valid version bump types are `patch`, `minor` and `major`.
-Please always include the `firebase` package with the same version bump type as
-your package. This is to ensure that the version of the `firebase` package will
-be bumped correctly. For example,
+    ```shell
+    yarn changeset
+    ```
 
-```
----
-"@firebase/storage": minor
-"firebase": minor
----
+You will be asked to create a description (here's an
+[example]((https://github.com/firebase/firebase-js-sdk/pull/3284#issuecomment-649718617)), select
+which Firebase product packages you’ve updated, and if the change is a major, minor or simply a
+patch revision, where:
 
-This is a test.
-```
+[Example message]).
+ * a major version is an incompatible API change
+ * a minor version is a backwards compatible API change
+ * a patch version is a backwards compatible bug fix or any change that does not affect the API. A
+   refactor, for example.
+
+You do not need to create a Changeset for the following changes:
+
+ * the addition or alteration of a test
+ * documentation updates
+ * updates to the repository’s CI
```

Comment:
Maybe TMI but while we are here, maybe add that if it's major or minor, you also have to add `'firebase': minor` (or major) as the changesets tool doesn't automatically do this for monorepos.

Reply:
It's only needed for minor or major but it doesn't hurt for patch, so maybe this is less confusing.